mkdocs.yml

```diff
@@ -122,6 +122,7 @@ pages:
         - Kubernetes Audit: integration/kubernetes-audit-integration.md
         - GitHub Webhook Events: integration/github-webhook-events-integration.md
         - Journald: integration/journald-integration.md
+        - Vercel: integration/vercel-logs-integration.md
```

Comment:
Why not just vercel.md ?  You think we'll have different types of Vercel integrations?

Reply:
Maybe metrics down the road?

Reply:
OK.  Wouldn't we put that in the same /docs/integration/vercel?  Ah, maybe not, because now we have Integrations > .... split into two sections.... OK, ignore this comment.